test/index.html

```diff
@@ -49,79 +49,146 @@
     .microsite {
       height: 300px;
     }
-  </style>
-</head>
-
-<body>
-  <h2>Nice org</h2>
-  <iframe src="./example.niceorg.html"></iframe>
-  <iframe src="./example.niceorg.html"></iframe>
-
-  <hr/>
-
-  <h2>Guidance</h2>
-  <iframe src="./example.guidance.html"></iframe>
-  <iframe src="./example.guidance.html"></iframe>
+    .dropbtn {
+      background-color: #126470;
+      color: white;
+      padding: 16px;
+      font-size: 16px;
+      border: none;
+      cursor: pointer;
+    }
 
-  <hr/>
+    .dropbtn:hover, .dropbtn:focus {
+      background-color: #004650;
+    }
 
-  <h2>Standards and Indicators</h2>
-  <iframe src="./example.standards.html"></iframe>
-  <iframe src="./example.standards.html"></iframe>
+    .dropdown {
+      position: relative;
+      display: inline-block;
+      background-color: #126470;
+      width: 100%;
+      height: 70px;
+      color: white
+    }
 
-  <hr/>
+    .dropdownContent {
+      display: none;
+      position: absolute;
+      background-color: #f9f9f9;
+      min-width: 160px;
+      overflow: auto;
+      box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+      z-index: 1;
+    }
 
-  <h2>Evidence</h2>
-  <iframe src="./example.evidence.html"></iframe>
-  <iframe src="./example.evidence.html"></iframe>
+    .dropdownContent a {
+      color: black;
+      padding: 12px 16px;
+      text-decoration: none;
+      display: block;
+    }
 
-  <hr/>
+    .dropdown a:hover {background-color: #f1f1f1}
 
-  <h2>Evidence search</h2>
-  <iframe src="./example.evidence-search.html"></iframe>
-  <iframe src="./example.evidence-search.html"></iframe>
+    .show {display:block;}
+    .hidden{display: none;}
+    #dropdownList{
+      height:  300px;
+      overflow: scroll;
+      overflow-x: hidden;
+    }
+    form{
+      display: inline;
+    }
 
-  <hr/>
 
-  <h2>BNF microsite</h2>
-  <iframe src="./example.bnf.html" class="microsite"></iframe>
-  <iframe src="./example.bnf.html"></iframe>
+  </style>
+  <script
+  src="https://code.jquery.com/jquery-3.1.1.js"
+  integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA="
+  crossorigin="anonymous"></script>
+  <script>
+    /* When the user clicks on the button, 
+    toggle between hiding and showing the dropdown content */
+    
+
+    function triggerDropdown() {
+      $("#dropdownList").toggleClass("show");
+    }
 
-  <hr/>
+// Close the dropdown if the user clicks outside of it
+$(window).click(function() {
+  if (!event.target.matches(".dropbtn")) {
```

Comment:
Try to use e as a parameter for the handler and using that as jquery abstracts this for different browers